pkgs/applications/misc/foxtrotgps/default.nix

```
@@ -0,0 +1,39 @@
+{ fetchurl, stdenv, pkg-config, wrapGAppsHook, curl, gnome2, gpsd, gtk2
+, intltool, libexif, python3Packages, sqlite }:
+
+stdenv.mkDerivation rec {
+  pname = "foxtrotgps";
+  version = "1.2.2";
+
+  src = fetchurl {
+    url = "https://www.foxtrotgps.org/releases/foxtrotgps-${version}.tar.xz";
+    sha256 = "0grn35j5kwc286dxx18fv32qa330xmalqliwy6zirxmj6dffvrkg";
+  };
+
+  nativeBuildInputs = [ pkg-config wrapGAppsHook ];
+
+  buildInputs = [
+    curl.dev
+    gnome2.libglade.dev
+    gpsd
+    gtk2.dev
+    intltool
+    libexif
+    sqlite.dev
+    (python3Packages.python.withPackages (pythonPackages: with python3Packages;
+    [ beautifulsoup4 feedparser sqlalchemy ]))
+    ];
+
+  meta = with stdenv.lib; {
+    description = ''
```

Maybe that would look better in the online pacakge list (https://nixos.org/nixos/packages.html)?
```suggestion
    description = "GPS/GIS application optimized for small screens"
    longDescription = ''
```